### PR TITLE
adding RTL support

### PR DESCRIPTION
**Pull request by iamsoorena**
I started using this library on my `RTL` react-native app and I found out that It crashes on `RTL`.
so I did some edits to make it work in `RTL` too.
I've tested my branch in both `LTR` and `RTL` apps and it works.

### CODE REVIEW

File: src/Menu.js [hunk 1]
```diff
@@ -122,16 +123,30 @@ class Menu extends React.Component {
     };
 
     // Adjust position of menu
-    let { left, top } = this.state;
+    let LEFT_OR_RIGHT = null;
+    let { top } = this.state;
+    
+    if (I18nManager.isRTL) {
```
Comment (mxck) [inline src/Menu.js]: You can just rename `left` and `top` to `x` and `y`, so you don't need create extra variables.

**Comment by mxck** [comment]
@iamsoorena you can just use new view prop type `start`. So you need only change `left` in style to `start` 💯

**Comment by iamsoorena** [comment]
@mxck WOW. the last one was hell of a tip.
I'll update this PR abd come back. 😄

**Comment by iamsoorena** [comment]
@mxck actually I ended up using `end` instread of `start`, now thanks to you with minimum change we support `RTL` languages too.

**Comment by mxck** [comment]
@iamsoorena why `end`? 🤔

**Comment by iamsoorena** [comment]
@mxck this is so strange because when I used `start` everything was flipped in a wrong way. let me check this out, I'll update you.

**Comment by iamsoorena** [comment]
@mxck I just double checked this.
if I use `start` instead of `end`, everything will be flipped in a wrong way.
the most important thing is. If I place the Menu to the right. expanded menu will show up on the left and vice versa. this thing only works for me If I use `end`. I know it's weird.
did you check the edges in `rtl` mode?

**Comment by mxck** [comment]
@iamsoorena sorry for long answer, I check it again later with last react-native version.

**Comment by alikazemkhanloo** [comment]
I also wanted to make this library RTL for my app, ended up with replacing `left` with `end`.

**Comment by mxck** [comment]
At the moment I can not test the application for RTL.
If there is a normal pull request with a description, I will accept it.

**Pull request closed without merging**